lib/usagereporter/teleport/audit.go

```diff
@@ -80,6 +81,26 @@ func ConvertAuditEvent(event apievents.AuditEvent) Anonymizable {
 			RequiredPrivateKeyPolicy: e.RequiredPrivateKeyPolicy,
 		}
 
+	case *apievents.AccessRequestCreate:
+		switch e.GetType() {
+		case events.AccessRequestCreateEvent:
+			if e.User == "" || e.User == teleport.UserSystem {
+				return nil
+			}
+			return &AccessRequestCreateEvent{
+				UserName: e.User,
+			}
+		case events.AccessRequestReviewEvent:
+			if e.Reviewer == "" || e.User == teleport.UserSystem {
```

Comment:
Should this be
```suggestion
			if e.Reviewer == "" || e.Reviewer == teleport.UserSystem {
```
?

Reply:
Yeah right. Thanks for catching that, fixed in https://github.com/gravitational/teleport/pull/52170/commits/ab393a607c9c7cbe6460fcb046bc98c2eaa127a4